t/03-proxy_wasm/003-on_tick.t

```diff
@@ -128,6 +128,7 @@ ok
 
 
 === TEST 5: proxy_wasm - multiple ticks
+--- disable_rust_backtrace
```

Comment:
Can't we update the error log regex and always have the backtrace details env directives instead?

Reply:
I'm not sure. It's not the error log regex that's conflicting with the backtrace, it seems to be the `stub`:
`pattern "stub" should not match any line in error.log but matches line "   7: wasmtime::trampoline::func::stub_fn" (req 0)`

It also seemed right suppressing the backtrace from the console output in a successful test run.